tests/testsuite/directory.rs

```diff
@@ -449,66 +449,3 @@ fn git_repo_in_home_directory_truncate_to_repo_true() -> io::Result<()> {
     assert_eq!(expected, actual);
     Ok(())
 }
-
-#[test]
-#[ignore]
-fn use_logical_and_physical_paths() -> io::Result<()> {
```

Comment:
This test is removed because

- `--path` argument will overwrite the logical path, rather than the physical path. The physical path cannot be tested as it will always be the directory where we run `starship` command.
- Testing logical path only looks meaningless since it just tests if the function reads `$PWD` correctly, as stated in the comment.